src/main/java/com/chillimport/server/FileManager.java

```diff
@@ -2,333 +2,325 @@
 
 import com.chillimport.server.errors.LogManager;
 import com.chillimport.server.utility.FileStorageException;
-import org.springframework.core.io.FileSystemResource;
-import org.springframework.core.io.Resource;
-import org.springframework.stereotype.Service;
-import org.springframework.util.StringUtils;
-import org.springframework.web.multipart.MultipartFile;
-
 import java.io.*;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.nio.file.*;
 import java.time.ZonedDateTime;
-
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import org.springframework.web.multipart.MultipartFile;
 
 @Service
 /**
- * This class manages all File interactions (saving, loading, deleting) and provides Paths for all files.
- * These include: Configurations, Logs and file uploads
+ * This class manages all File interactions (saving, loading, deleting) and
+ * provides Paths for all files. These include: Configurations, Logs and file
+ * uploads
  *
  * It also holds the server URL and login credentials
  */
 public class FileManager {
 
-    private static Path CONFIG_PATH = null;
-    private static Path BASE_PATH = null;
-    private static Path LOG_PATH = null;
-    private static Path FILES_PATH = null;
-    private static String USER_NAME_PASSWORD = null;
-    private static String sep = File.separator;
-
-    public FileManager() {
+  private static Path CONFIG_PATH = null;
+  private static Path BASE_PATH = null;
+  private static Path LOG_PATH = null;
+  private static Path FILES_PATH = null;
+  private static String USER_NAME_PASSWORD = null;
+  private static String sep = File.separator;
+
+  public FileManager() {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=5167523)